eth/db/chain.py

```diff
@@ -239,8 +139,8 @@ def persist_block(self,
     @classmethod
     def _persist_block(
             cls,
-            db: 'BaseDB',
```

Comment:
This resulted in a lot of cleanups like this where we no longer have circular import issues with type hints.